Applications/WebPortal/WebApp/library/source/pages/account_directory_page/account_directory_page.tsx

```diff
@@ -105,6 +116,21 @@ export class AccountDirectoryPage extends React.Component<Properties, State> {
 
   private onChange(newFilter: string) {
     this.setState({filter: newFilter});
+    this.setState({
+      openedGroups: new Beam.Set<Beam.DirectoryEntry>()
+    });
+  }
+
+  private async onCardClick(group: Beam.DirectoryEntry) {
+    if(!this.state.accounts.get(group)) {
```

Comment:
Always perform the load no matter what rather than only loading if you never loaded before.